docker/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3'
```

Comment:
If this docker compose file is only used for bookkeeper testing, please move these files to `test/docker/bookkeeper`.